nlp/semanticpackage/.htaccess

```diff
@@ -0,0 +1,8 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://gplsi.dlsi.ua.es/gplsi13/sites/default/files/resources/semanticpackage.owl [R=302,L]
+RewriteRule ^_1.1$ https://gplsi.dlsi.ua.es/gplsi13/sites/default/files/resources/semanticpackage_1.1.owl [R=302,L]
+RewriteRule ^_bucket_BarcelonaOpen2015$ https://gplsi.dlsi.ua.es/gplsi13/sites/default/files/resources/semanticpackage_bucket_BarcelonaOpen2015.owl [R=302,L]
```

Comment:
I don't think these do what you mean to do.  They will match paths like `/nlp/semanticpackage/_bucket_BarcelonaOpen2015`.  Yet in README you list `/nlp/semantic_bucket_BarcelonaOpen2015`.